### PR TITLE
hide add button if user d'ont have create content role

### PR DESCRIPTION
**Pull request by djamnazi**
[OO-BUGFIX] hide add button if user d'ont have create content role

https://github.com/open-orchestra/open-orchestra-cms-bundle/issues/1690

### CODE REVIEW

File: ApiBundle/Transformer/ContentCollectionTransformer.php [hunk 1]
```diff
@@ -30,7 +31,7 @@ public function transform($contentCollection, $contentType = null)
             array()
         ));
 
-        if ($contentType) {
+        if ($contentType && $this->authorizationChecker->isGranted(ContentTypeForContentPanelStrategy::ROLE_ACCESS_CREATE_CONTENT_TYPE_FOR_CONTENT)) {
```
Comment (itkg-ngilain) [inline ApiBundle/Transformer/ContentCollectionTransformer.php]: I'm not sure you check the correct right: I thought the issue was on the content creation. The ROLE_ACCESS_CREATE_CONTENT_TYPE_FOR_CONTENT should be used to allow the creation of content type, not content

Comment (djamnazi) [inline ApiBundle/Transformer/ContentCollectionTransformer.php]: @itkg-ngilain  i check the correct right, this is the same right used for the NEW ACTION in contentController. The right for create content type is: ROLE_ACCESS_CREATE_CONTENT_TYPE. you can verify this in the contentTypeController.

Comment (itkg-ngilain) [inline ApiBundle/Transformer/ContentCollectionTransformer.php]: @djamnazi : allright so, the naming is ambiguous, but it's not due to your PR.

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**